pkg/api/vlabs/validate.go

```diff
@@ -295,8 +295,13 @@ func (o *OpenShiftConfig) Validate() error {
 
 // Validate implements APIObject
 func (m *MasterProfile) Validate(o *OrchestratorProfile) error {
-	if o.OrchestratorType == OpenShift && m.Count != 1 {
-		return errors.New("openshift can only deployed with one master")
+	if o.OrchestratorType == OpenShift {
+		if m.Count != 1 {
+			return errors.New("openshift can only deployed with one master")
+		}
+		if (m.VnetSubnetID != "" && m.FirstConsecutiveStaticIP == "") || (m.FirstConsecutiveStaticIP != "" && m.VnetSubnetID == "") {
+			return errors.New("when deploying openshift using an existing vnet the vnetSubnetId and firstConsecutiveStaticIP are required")
```

Comment:
if m.VnetSubnetID == "" then we are not deploying with an an existing vnet, right?

Reply:
@kargakis,  @jim-minter and I chatted about this.  We would expect the vnetsubnetid and the firstconsecutivestaticip to be defined if deploying into an existing vnet.

@jim-minter, does this sound correct?

Reply:
@kwoodson the error string needs to be reworded or maybe become 2 error strings - i.e. "if you specify an existing vnet you must specify the ip" and "you can't specify the ip unless you specify an existing vnet"

Reply:
Ok, will break into 2 separate statements.